apps/daimo-mobile/src/sync/sync.ts

```diff
@@ -301,39 +307,15 @@ function addNamedAccounts(old: EAccount[], found: EAccount[]): EAccount[] {
 
 /** Add transfers based on new Transfer event logs */
 function addTransfers(
-  old: TransferOpEvent[],
-  logs: TransferOpEvent[],
-  daimoChain: DaimoChain
-): TransferOpEvent[] {
-  // Start with old, finalized transfers
-  const ret = [...old];
-
+  old: DisplayOpEvent[],
+  logs: DisplayOpEvent[]
+): DisplayOpEvent[] {
   // Sort new logs
   logs.sort((a, b) => {
     if (a.blockNumber !== b.blockNumber) return a.blockNumber! - b.blockNumber!;
     return a.logIndex! - b.logIndex!;
   });
 
-  // Add new transfers since previous lastFinalizedBlock
-  for (const transfer of logs) {
-    ret.push({
-      type: "transfer",
-      status: OpStatus.confirmed,
-
-      from: transfer.from,
-      to: transfer.to,
-      amount: Number(transfer.amount),
-      nonceMetadata: transfer.nonceMetadata,
-
-      timestamp: guessTimestampFromNum(transfer.blockNumber!, daimoChain),
-      txHash: transfer.txHash,
-      blockNumber: transfer.blockNumber,
-      blockHash: transfer.blockHash,
-      logIndex: transfer.logIndex,
-      opHash: transfer.opHash,
-      feeAmount: transfer.feeAmount,
-    });
-  }
-
-  return ret;
```

Comment:
If we change the return type of `getAccountHistory`, how do we avoid breaking existing apps?